test/MUXControlsReleaseTest/NugetPackageTestAppCX/NugetPackageTestAppCX.vcxproj

```diff
@@ -194,6 +195,27 @@
       <Project>{e7d6b510-3761-478a-8e22-72558dad924e}</Project>
     </ProjectReference>
   </ItemGroup>
+  <!-- Include pgort140.dll in the app package -->
+  <ItemGroup Condition="'$(Configuration)' == 'Release' and '$(PGOBuildMode)' == 'Instrument'">
+    <None Include="$(PGOFilesDir)\pgort140.dll">
+      <Link>%(Filename)%(Extension)</Link>
+      <DeploymentContent>true</DeploymentContent>
+      <Visible>false</Visible>
+      <InProject>false</InProject>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+	  <None Include="$(PGOFilesDir)\pgosweep.exe">
+      <Visible>false</Visible>
+      <InProject>false</InProject>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+	  <None Include="$(PGOFilesDir)\vcruntime140.dll" Condition="Exists('$(PGOFilesDir)\vcruntime140.dll')">
```

Comment:
I am surprised that this is necessary, since the app uses the VCLibs framework package. Do you know why this is necessary?

Reply:
the tools seem to need these in the same folder.

---
In reply to: [301745225](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r301745225) [](ancestors = 301745225)